README.md

```diff
@@ -1 +1,31 @@
-# googleapi1
\ No newline at end of file
+Installation & Usage
+
+Installation
+
+Clone or download the repo.
+Open terminal and navigate to Google folder.
+Run npm install to install dependencies.
+
+Usage
+Navigate to server.
+Run nodemon app.js to launch the server.
+Navigate to index.html in client then open with live server.
+
+Tests
+Run npm test to launch test suite(in progress).
+
+Purpose
+An API that can be used to search key words and return a number of premier league teams and their information.
```

Comment:
Purpose is really clear! Maybe some examples of what you can put in the search bar would be helpful for first-time users :)